analyzers/its/regression-test.ps1

```diff
@@ -312,7 +307,7 @@ function IssuesAreEqual($actual, $expected){
 function VerifyUnexpectedIssues($actualIssues, $expectedIssues){
     $unexpectedIssues = @()
 
-    foreach ($actualIssue in $actualIssues){
+    foreach ($actualIssue in $actualIssues | Where-Object { $ruleId -eq "" -or $_.IssueId -eq $ruleId }){
```

Comment:
Hotspots are executed, as per `SonarLint.xml` configuration` so we need to filter them out for comparing when `$ruleId` is set.

Fixing the root cause is another effort (I'll keep a note)